scripts/e2e_test_keras.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [[ $# -ne 6 ]] ; then
+  echo "Usage: $0 <path/to/keras/model> <path/to/emitc/include/files> <path/to/emitc-opt> <compiler> <seed> <output_dir>"
+  echo
+  echo "This script expects a python version in the PATH with a recent version of tensorflow installed."
+  echo "Tested with python 3.6.9 and tensorflow 2.4.0" 
+
+  exit 1
+fi
+
+KERAS_MODEL=$1
+EMITC_INCLUDE_DIR=$2
+EMITC_OPT=$3
+EMITC_TRANSLATE=$(dirname $EMITC_OPT)/emitc-translate
+CPP_COMPILER=$4
+SEED=$5
+BATCH_SIZE=2
```

Comment:
Should we make this an input parameter? In internal use, a colleague hit the issue that he expected a batch size of 1. This can be a followup though.

Reply:
Done.